Documentation/getting-started.md

```diff
@@ -22,6 +22,17 @@ A graphics card that supports OpenCL 1.2 or higher is required to run Autoscoper
 If you are using Autoscoper on a remote computer, you will likely need to setup a [remote GPU](./adv-topics/remote-gpu-setup.md) to enable GPU acceleration.
 ```
 
+```{important}
+The AutoscoperM module’s frame-level tracking executable requires a supported GPU rendering backend and is therefore available **only** on Linux and Windows. macOS is not supported due to rendering backend limitations.
+
+| Module                       | Linux | macOS | Windows |
+|------------------------------|-------|-------|---------|
+| AutoscoperM (CUDA)           | ❌    | ❌    | ✅       |
+| AutoscoperM (OpenCL)         | ✅    | ❌    | ✅       |
```

Comment:
The Pre-processing tab in the AutoscoperM module works for all platforms, which can be useful for segmentation and model generation before using the 3D registration model. Should we clarify this here somehow?